### DISCUSSION TITLE
Customize Cypress Browser Profile

### DISCUSSION THREAD

**Discussion by EtienneBruines** [cypress-io/cypress#15982] [Questions and Help]
I´ve been running multiple Cypress instances in parallel (so instead of using puppeteer to simulate the second browser, using Cypress to achieve that).

The issue that I´ve been having lately (since about `6.5.0` or so), is that both instances share the same storage, meaning that once I clear IndexedDB in instance A, the IndexedDB is emptied in instance B as well.

Question: (how) can I specify the path of the browser-profile to use? 

At the [Launching Browsers](https://docs.cypress.io/guides/guides/launching-browsers#Cypress-Profile) page it mentions that Cypress has its own custom profile -- I was wondering if there's a way to specify it. `before:browser:launch` did not yield any information to the path. 

----

What hasn´t worked:

```js
module.exports = (on, config) => {
  on('before:run', (details) => {
    details.browser.profilePath = '/tmp/profile-1'
  })
  on('before:browser:launch', (browser, launchOptions) => {
    browser.profilePath = '/tmp/profile-1'
  })
}
```

**Comment by EtienneBruines** [comment]
I ended up specifying the `HOME` environment variable; that way, Cypress creates all nice subfolders for caching and profile directories.

**Reply by sorenhansendk** [reply]
You can set the `XDG_CONFIG_HOME` to change the path, so you don't need to change the `HOME` directory.
